e2e/test/scenarios/dashboard-filters/dashboard-filters-query-stages.cy.spec.ts

```diff
@@ -776,87 +968,271 @@ describe("scenarios > dashboard > filters > query stages", () => {
-          ["User", PEOPLE_DATE_COLUMNS],
-          ["Summaries", ["Created At: Month", "Created At: Year"]],
-        ]);
-        verifyNoDashcardMappingOptions(QUESTION_BASED_MODEL_INDEX);
-        verifyNoDashcardMappingOptions(MODEL_BASED_MODEL_INDEX);
-      }
+        function verifyDateMappingOptions() {
+          verifyDashcardMappingOptions(QUESTION_BASED_QUESTION_INDEX, [
+            ["Base Orders Question", ORDERS_DATE_COLUMNS],
+            ["Reviews", [...REVIEWS_DATE_COLUMNS, ...REVIEWS_DATE_COLUMNS]], // https://github.com/metabase/metabase/issues/46845
+            ["Product", [...PRODUCTS_DATE_COLUMNS, ...PRODUCTS_DATE_COLUMNS]], // https://github.com/metabase/metabase/issues/46845
+            ["User", PEOPLE_DATE_COLUMNS],
+            ["Summaries", ["Created At: Month", "Created At: Year"]],
+          ]);
+          verifyDashcardMappingOptions(MODEL_BASED_QUESTION_INDEX, [
+            ["Base Orders Model", ORDERS_DATE_COLUMNS],
+            ["Reviews", [...REVIEWS_DATE_COLUMNS, ...REVIEWS_DATE_COLUMNS]], // https://github.com/metabase/metabase/issues/46845
+            ["Product", [...PRODUCTS_DATE_COLUMNS, ...PRODUCTS_DATE_COLUMNS]], // https://github.com/metabase/metabase/issues/46845
+            ["User", PEOPLE_DATE_COLUMNS],
+            ["Summaries", ["Created At: Month", "Created At: Year"]],
+          ]);
+          verifyNoDashcardMappingOptions(QUESTION_BASED_MODEL_INDEX);
+          verifyNoDashcardMappingOptions(MODEL_BASED_MODEL_INDEX);
+        }
 
-      function verifyTextMappingOptions() {
-        verifyDashcardMappingOptions(QUESTION_BASED_QUESTION_INDEX, [
-          ["Reviews", [...REVIEWS_TEXT_COLUMNS, ...REVIEWS_TEXT_COLUMNS]],
-          ["Product", [...PRODUCTS_TEXT_COLUMNS, ...PRODUCTS_TEXT_COLUMNS]], // https://github.com/metabase/metabase/issues/46845
-          ["User", PEOPLE_TEXT_COLUMNS],
-          ["Summaries", ["Category"]],
-        ]);
-        verifyDashcardMappingOptions(MODEL_BASED_QUESTION_INDEX, [
-          ["Reviews", [...REVIEWS_TEXT_COLUMNS, ...REVIEWS_TEXT_COLUMNS]],
-          ["Product", [...PRODUCTS_TEXT_COLUMNS, ...PRODUCTS_TEXT_COLUMNS]], // https://github.com/metabase/metabase/issues/46845
-          ["User", PEOPLE_TEXT_COLUMNS],
-          ["Summaries", ["Category"]],
-        ]);
-        verifyDashcardMappingOptions(QUESTION_BASED_MODEL_INDEX, [
-          [
-            NAMELESS_SECTION,
+        function verifyTextMappingOptions() {
+          verifyDashcardMappingOptions(QUESTION_BASED_QUESTION_INDEX, [
+            ["Reviews", [...REVIEWS_TEXT_COLUMNS, ...REVIEWS_TEXT_COLUMNS]],
+            ["Product", [...PRODUCTS_TEXT_COLUMNS, ...PRODUCTS_TEXT_COLUMNS]], // https://github.com/metabase/metabase/issues/46845
+            ["User", PEOPLE_TEXT_COLUMNS],
+            ["Summaries", ["Category"]],
+            // ["Summaries (2)", ["Reviewer", "Category"]], // https://github.com/metabase/metabase/issues/48339
```

Comment:
Can you add `TODO`'s to unfixed issue links?

Reply:
c447b675d47f77292509ae3376d7425ebc979726